internal/network/command_transit_gateway_attachment.go

```diff
@@ -104,27 +106,27 @@ func printTransitGatewayAttachmentTable(cmd *cobra.Command, attachment networkin
-			NetworkId:                  attachment.Spec.Network.GetId(),
-			RamShareArn:                attachment.Spec.Cloud.NetworkingV1AwsTransitGatewayAttachment.GetRamShareArn(),
-			TransitGatewayId:           attachment.Spec.Cloud.NetworkingV1AwsTransitGatewayAttachment.GetTransitGatewayId(),
-			Routes:                     strings.Join(attachment.Spec.Cloud.NetworkingV1AwsTransitGatewayAttachment.GetRoutes(), ", "),
-			TransitGatewayAttachmentId: attachment.Status.Cloud.NetworkingV1AwsTransitGatewayAttachmentStatus.GetTransitGatewayAttachmentId(),
-			Phase:                      attachment.Status.GetPhase(),
+			Id:                            attachment.GetId(),
+			Name:                          attachment.Spec.GetDisplayName(),
+			NetworkId:                     attachment.Spec.Network.GetId(),
+			AwsRamShareArn:                attachment.Spec.Cloud.NetworkingV1AwsTransitGatewayAttachment.GetRamShareArn(),
+			AwsTransitGatewayId:           attachment.Spec.Cloud.NetworkingV1AwsTransitGatewayAttachment.GetTransitGatewayId(),
+			Routes:                        strings.Join(attachment.Spec.Cloud.NetworkingV1AwsTransitGatewayAttachment.GetRoutes(), ", "),
+			AwsTransitGatewayAttachmentId: attachment.Status.Cloud.NetworkingV1AwsTransitGatewayAttachmentStatus.GetTransitGatewayAttachmentId(),
+			Phase:                         attachment.Status.GetPhase(),
 		})
 	} else {
 		table.Add(&transitGatewayAttachmentSerializedOut{
-			Id:                         attachment.GetId(),
-			Name:                       attachment.Spec.GetDisplayName(),
-			NetworkId:                  attachment.Spec.Network.GetId(),
-			RamShareArn:                attachment.Spec.Cloud.NetworkingV1AwsTransitGatewayAttachment.GetRamShareArn(),
-			TransitGatewayId:           attachment.Spec.Cloud.NetworkingV1AwsTransitGatewayAttachment.GetTransitGatewayId(),
-			Routes:                     attachment.Spec.Cloud.NetworkingV1AwsTransitGatewayAttachment.GetRoutes(),
-			TransitGatewayAttachmentId: attachment.Status.Cloud.NetworkingV1AwsTransitGatewayAttachmentStatus.GetTransitGatewayAttachmentId(),
-			Phase:                      attachment.Status.GetPhase(),
+			Id:                            attachment.GetId(),
+			Name:                          attachment.Spec.GetDisplayName(),
+			NetworkId:                     attachment.Spec.Network.GetId(),
+			AwsRamShareArn:                attachment.Spec.Cloud.NetworkingV1AwsTransitGatewayAttachment.GetRamShareArn(),
+			AwsTransitGatewayId:           attachment.Spec.Cloud.NetworkingV1AwsTransitGatewayAttachment.GetTransitGatewayId(),
+			Routes:                        attachment.Spec.Cloud.NetworkingV1AwsTransitGatewayAttachment.GetRoutes(),
+			AwsTransitGatewayAttachmentId: attachment.Status.Cloud.NetworkingV1AwsTransitGatewayAttachmentStatus.GetTransitGatewayAttachmentId(),
+			Phase:                         attachment.Status.GetPhase(),
 		})
 	}
 
-	return table.Print()
+	return table.PrintWithAutoWrap(false)
```

Comment:
@brianstrauch is it fine if i use this? otherwise the output looks like this
![image](https://github.com/confluentinc/cli/assets/135164550/c05ca050-6bdb-4617-9e6c-7c78f530ed91)

Reply:
Yeah, that's fine... but what if we just remove the trailing "ID"... It's clear from looking at it that it's an ID.

Reply:
Likewise, we can just say "AWS Transit Gateway" and "Network"... up to you!

Reply:
Thank you! I think we can keep the `ID` here to keep it consistent with our other `network` subcommands.